theme/templates/resource-landing-page/title-section.html

```diff
@@ -173,52 +190,138 @@ <h2 id="resource-title">{{ title }}</h2>
               method="post">
             {% csrf_token %}
             <fieldset>
                 <div id="div_id_value" class="control-group">
-                    <div class="controls"><input class="form-control input-sm textinput textInput"
-                                                 id="txt-title"
-                                                 maxlength="300" name="value" type="text"
-                                                 value="{{ cm.metadata.title }}"></div>
+                    <div class="controls">
+                        <input class="form-control input-sm textinput textInput" id="txt-title"
+                                {% if communities_enabled and czo_user %} onclick="titleClick()" {% endif %}
+                               maxlength="300" name="value" type="text" value="{{ cm.metadata.title }}"/>
+                    </div>
                 </div>
                 <div>
-                    <button type="button" class="btn btn-primary pull-right btn-form-submit"
+                    <button type="button" id="title-save-button" class="btn btn-primary pull-right btn-form-submit"
                             style="display: none;">
                         Save changes
                     </button>
                 </div>
             </fieldset>
         </form>
+
     {% endif %}
-</div>
 
 
 {% if is_replaced_by %}
     <div class="col-xs-12">
         <div class="clear-fix"><i>A newer version of this resource <a
-                href="{{ is_replaced_by }}">{{ is_replaced_by }}</a> is available that replaces this version.</i></div>
+                href="{{ is_replaced_by }}">{{ is_replaced_by }}</a> is available that replaces this version.</i>
+        </div>
     </div>
 {% endif %}
 {% if is_version_of %}
     <div class="col-xs-12">
         <div class="clear-fix"><i>An older version of this resource <a
-                href="{{ is_version_of }}">{{ is_version_of }}</a> is available.</i></div>
+            href="{{ is_version_of }}">{{ is_version_of }}</a> is available.</i>
+        </div>
     </div>
 {% endif %}
 
 {% if cm.resource_type == "ToolResource" and cm.metadata.approved %}
     <div class="col-xs-12">
-        <div class="clear-fix"><img class="app-approved-icon-img" src="/static/img/approved_app.png"><i>This WebApp resource has been approved by HydroShare admin.</i></div>
+        <div class="clear-fix"><img class="app-approved-icon-img" src="/static/img/approved_app.png"><i>This WebApp
+            resource has been approved by HydroShare admin.</i>
+        </div>
     </div>
 {% endif %}
 
+{% if communities_enabled %}
+    {% block extra_js %}
+        <link rel="stylesheet" href="{{ STATIC_URL }}css/vue-mover.css">
+
+        <style>
+            [v-cloak] {
+                display: none
+            }
+
+            .title-builder-input {
+                margin-top: 1em;
+            }
+        </style>
+        <script src="{{ STATIC_URL }}js/Sortable.min.js"></script>
```

Comment:
Could you indicate where this script is being used? Sortable.min.js

Reply:
Eventually webpack, node or other frameworks should be introduced to manage this sort of thing.

Reply:
Fixed in 81118c7c7